README.md

```diff
@@ -33,6 +33,20 @@ To run the tests, execute the following commands:
 
     bundle exec rake
 
+## Linting and formatting
+
+Rubocop is used for linting.
+
+```shell
+bundle exec rubocop
+```
+
+Prettier is used for formatting.
+
+```shell
+bundle exec rbprettier --check . --ignore-unknown
```

Comment:
I didnt know about this

Reply:
Yeah its a bit hidden, I only found it because its used in the CI.